moveit_setup_assistant/templates/moveit_config_pkg_template/launch/pilz_industrial_motion_planner_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,14 @@
+<launch>
+
+  <!-- Pilz Command Planner Plugin for MoveIt! -->
+  <arg name="planning_plugin" value="pilz_industrial_motion_planner::CommandPlanner" />
+
+  <arg name="planning_adapters" value="" />
+
+  <arg name="start_state_max_bounds_error" value="0.1" />
+
+  <param name="planning_plugin" value="$(arg planning_plugin)" />
+  <param name="request_adapters" value="$(arg planning_adapters)" />
+  <param name="start_state_max_bounds_error" value="$(arg start_state_max_bounds_error)" />
+
```

Comment:
Shouldn't this have the new capabilities, like [here](https://github.com/ros-planning/moveit_resources/blob/2d207b4de949c9e4ff80acfd07004133bd054a39/panda_moveit_config/launch/pilz_industrial_motion_planner_planning_pipeline.launch.xml#L15)?

Reply:
Yep, absolutely. To allow passing in user-defined capabilities as well, I'd suggest to pass the `capabilities` argument down to the included pipeline launch file(s) and append the planner-specific ones there: 99c059f

I'll re-generate the configuration inside moveit_resources based on this template.